src/app/static/src/app/components/header/OnlineSupportMenu.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <drop-down text="support" :right="true" style="flex: none">
+        <a class="dropdown-item"
+           :href="'https://mrc-ide.github.io/naomi-troubleshooting/' + troubleFilename"
+           target="_blank"> FAQ
+        </a>
+        <a class="dropdown-item"
+           :href="urls"
+           target="_blank"> Contact
+        </a>
+    </drop-down>
+</template>
+
+<script lang="ts">
+    import Vue from "vue";
+    import DropDown from "./DropDown.vue";
+    import i18next from "i18next";
+    import {mapStateProp} from "../../utils";
+    import {RootState} from "../../root";
+    import {Language} from "../../store/translations/locales";
+    import {switches} from "../../featureSwitches";
+
+    interface Props {
+        troubleFilename: string
+    }
+
+    interface Computed {
+        support: string
+        urls: string
+        currentLanguage: Language
+    }
+
+    export default Vue.extend<unknown, unknown, Computed, Props>({
+        props: {
+            troubleFilename: String
+        },
+        computed: {
+            currentLanguage: mapStateProp<RootState, Language>(null,
+                (state: RootState) => state.language),
+            support() {
+                return i18next.t("support", this.currentLanguage)
+            },
+            urls() {
```

Comment:
Could we maybe rename this bugReportUrl to distinguish it from the other url? 'Urls' sounds like it's going to return more than one.